src/app/components/elements/AuthorDropdown.jsx

```diff
@@ -16,7 +16,7 @@ const AuthorDropdown = (props) => {
             <div className = "Author__container">
                 <div className="Author__dropdown">
                     <Link to={'/@' + props.author}>
-                        <Userpic account={props.author} width="75" height="75" />
```

Comment:
IIRC - main reason for passing size to Userpic is because it needs to know if it should request hi- or lo-res img from the server

Reply:
I see.. this functionality is removed entirely. Hmm. The high resolution image is bound to 320x320, whereas low resolution image is 120x120. Assuming the source image is a square, the high-res image will be ~7x bigger. My concern is that on some pages with 10's or 100's of (small) avatars this could really add up.

Reply:
With Userpic rendering only urls from `/u/${username}/avatar` we don't need to worry about sizing, as [Imagehoster takes care of it](https://github.com/steemit/imagehoster/pull/36/files#diff-ef4e02d1f3af008c9df50485227330ccR18).

Reply:
Imagehoster only serves 128x128 avatars as it is now

Reply:
It's a nice compromise size. A visual check of the only place the larger image was used looked good.

Also since the /avatar urls may point @ different images, we don't really want different sizes of profile image being cached for indefinitely long periods of time.

Reply:
We may want to bump it up to 150x150 or so. Existing sizes were chosen w/ retina sizes and some room for growth in mind. I think currently the max size we render is 75x75, so to look good on retina should 2x that.

A single size should work for now... but we'll need to revisit this if we ever need anything bigger than small or med-small image. Ideally the imagehoster would accept size params.

Reply:
Imagehoster accepting size params or a size profile (sm/med/lg/full) for avatars is a great idea.

Reply:
@roadscape - [I've created an issue to track sizing](https://github.com/steemit/imagehoster/issues/39). Please comment there if you think it could use revision. :)

Reply:
Ok, maybe we should keep the width/height param handling intact in Userpic if we want to leverage it later